crawl-ref/source/status.cc

```diff
@@ -962,6 +962,12 @@ static void _describe_gem(status_info& inf)
     if (time_taken >= limit)
         return; // already lost...
 
+    if (!gem_clock_active())
+    {
+        // player has picked up the orb, but the gem has not yet shattered
+        inf.light_text = "Gem not acquired";
+        return;
```

Comment:
This status text is a bit long - perhaps "No Gem" or "Gem*" or something like that could work. You should also assign `inf.light_colour` as you suggest, otherwise the status will show up in black text (which is invisible) - white is probably an appropriate colour, although something like cyan might make it stand out a bit more.